app/marketing/views.py

```diff
@@ -885,6 +886,25 @@ def trending_quests():
         ).order_by('-recent_attempts').all()[0:10]
     return quests
 
+def quest_of_the_day():
+    quest = trending_quests()[0]
+    return quest
+
+def upcoming_grant():
+    cutoff_date = timezone.now() - timezone.timedelta(days=7)
+    grant = Grant.objects.filter(created_on__gte=cutoff_date).order_by('-created_on')[:5][0]
```

Comment:
you should pick a grant that has gotten a contribution recently. not just the most recent grant

Reply:
So the grant that has got the latest contribution?

Reply:
Will this do?:
```
def upcoming_grant():
    grant = Contribution.objects.latest('created_on').subscription.grant
    return grant
```

Reply:
i think thats directionally right.  though a better approach might be to look at which grants have gotten the most contributions in the last month..  you can do this by ordering by `-weighted_shuffle`

Reply:
Done in [e9e4929](https://github.com/gitcoinco/web/pull/6471/commits/e9e4929592a8ae7d641218e74d5905844f794029)